cpp/ql/lib/semmle/code/cpp/ir/dataflow/internal/TaintTrackingUtil.qll

```diff
@@ -37,6 +38,13 @@ predicate localAdditionalTaintStep(DataFlow::Node nodeFrom, DataFlow::Node nodeT
   )
   or
   any(Ssa::Indirection ind).isAdditionalTaintStep(nodeFrom, nodeTo)
+  or
+  // object->field conflation for content that is a `TaintInheritingContent`.
```

Comment:
It would be nice if this comment expressed the intended direction, i.e. from qualifier to field in a read of `qualifier->field`.

Reply:
That was indeed what I thought I was communicating with the `object->field` arrow. But if that's not clear I can make it more explicit

Reply:
I see what you mean, for me the word "conflation" suggests a two-way relationship (merging two things into one) which threw me off a bit - but the comment in the `TaintInheritingContent` class itself is completely clear.